lisa/tools/wget.py

```diff
@@ -44,6 +44,12 @@ def get(
         matched_result = self.__pattern_path.match(command_result.stdout)
         if matched_result:
             download_file_path = matched_result.group("path")
+        else:
+            raise LisaException(
+                f"cannot find file path in output {command_result.stdout} of"
```

Comment:
add quota on variable outputs or put it at end of message, it can help reading. if it's too long, put it at end of message.

f"cannot find file path in stdout of '{run_command}', it may cause by download failed or pattern mismatch. stdout: {command_result.stdout}"